tests/testsuite.src/run_misc.at

```diff
@@ -4285,6 +4285,57 @@ AT_CHECK([COB_LOAD_CASE=UPPER ./caller], [0], [], [])
 
 AT_CLEANUP
 
+AT_SETUP([COB_LOAD_GLOBAL=true CALL/CANCEL works])
+AT_KEYWORDS([runmisc CALL CANCEL COB_LOAD_GLOBAL])
+
+AT_DATA([CALLEE.cob], [
+       IDENTIFICATION   DIVISION.
+       PROGRAM-ID.      callee.
+       PROCEDURE        DIVISION.
+           EXIT PROGRAM.
+])
+
+AT_DATA([caller.cob], [
+       IDENTIFICATION   DIVISION.
+       PROGRAM-ID.      caller.
+       PROCEDURE        DIVISION.
+           CALL "callee"
+           END-CALL.
+           CANCEL "callee".
+           STOP RUN.
+])
+
+AT_CHECK([$COMPILE_MODULE CALLEE.cob], [0], [], [])
+AT_CHECK([$COMPILE caller.cob], [0], [], [])
+AT_CHECK([COB_LOAD_GLOBAL=true ./caller], [0], [], [])
+
+AT_CLEANUP
+
+AT_SETUP([COB_LOAD_GLOBAL=false CALL/CANCEL still works])
+AT_KEYWORDS([runmisc CALL CANCEL COB_LOAD_GLOBAL])
+
+AT_DATA([CALLEE.cob], [
+       IDENTIFICATION   DIVISION.
+       PROGRAM-ID.      callee.
+       PROCEDURE        DIVISION.
+           EXIT PROGRAM.
+])
+
+AT_DATA([caller.cob], [
+       IDENTIFICATION   DIVISION.
+       PROGRAM-ID.      caller.
+       PROCEDURE        DIVISION.
+           CALL "callee"
+           END-CALL.
+           CANCEL "callee".
+           STOP RUN.
+])
+
+AT_CHECK([$COMPILE_MODULE CALLEE.cob], [0], [], [])
+AT_CHECK([$COMPILE caller.cob], [0], [], [])
```

Comment:
After initial testing - please combine those tests into one, possibly named "CALL/CANCEL and  COB_LOAD_GLOBAL"